_components/accordions.md

```diff
@@ -4,9 +4,153 @@ type: component
 title: Accordions
 ---
 
-<div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Accordions_UI_v1.png">
+<h3>Bordered Accordion</h3>
+
+<div class="code-sample">
+  <ul id="accordion-menu">
```

Comment:
Right on. Thanks to the UX team for your feedback and letting us know the best way to move forward :+1: 

@msecret feel free to change this to a class and update the JavaScript to reflect this change.